src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -15,16 +15,16 @@
 #include <ucs/sys/string.h>
 
 
-static UCS_F_ALWAYS_INLINE
+UCS_F_ALWAYS_INLINE
```

Comment:
why removed static?

Reply:
because this function is now also called from the tcp_listener.c file

Reply:
so need to move it to H file a static inline

Reply:
with the implementation you mean, right?

Reply:
yes